src/lib.rs

```diff
@@ -4023,6 +4039,51 @@ impl AsmFileExt {
     }
 }
 
+/// Returns true if `cc` has been disabled by `CC_FORCE_DISABLE`.
+fn is_disabled() -> bool {
+    static CACHE: AtomicU8 = AtomicU8::new(0);
+
+    let val = CACHE.load(Relaxed);
+    // We manually cache the environment var, since we need it in some places
+    // where we don't have access to a `Build` instance.
+    #[allow(clippy::disallowed_methods)]
+    fn compute_is_disabled() -> bool {
+        match std::env::var_os("CC_FORCE_DISABLE") {
+            // Not set? Not disabled.
+            None => false,
+            // Respect `CC_FORCE_DISABLE=0` and some simple synonyms.
+            Some(v) if &*v != "0" && &*v != "false" && &*v != "no" => false,
```

Comment:
This check appears to be the wrong way around (i.e. `CC_FORCE_DISABLE=0` returns `true` and `CC_FORCE_DISABLE=1` returns `false`).

Reply:
Thanks, I've opened https://github.com/rust-lang/cc-rs/pull/1371